pkg/csi_driver/controller_test.go

```diff
@@ -1678,6 +1735,7 @@ func TestCreateSnapshot(t *testing.T) {
 				Name:           backupName,
 				Parameters: map[string]string{
 					util.VolumeSnapshotTypeKey: "backup",
+					ParameterKeyLabels:         "key1:value1",
```

Comment:
why do we need this Line?

Reply:
I had injected the parameter to check a scenario in the existing test case. I have updated it by adding a new test case for label scenario.